src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
+
+.icon {
+  display: block;
+}
+
+.search-bar {
+  width: 100%;
+  margin-top: 20px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  overflow: hidden;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  position: relative;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bar:focus {
+  background-color: #f6f6f7;
+}
+
+.big-search-bar {
+  height: 70px;
+}
+
+.small-search-bar {
+  height: 42px;
+}
+
+.search-text {
```

Comment:
Add styles for the `::placeholder` as well 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/9798aad2-1959-4b2f-a10f-1684d5b01e7a)